pages/common/stress-ng.md

```diff
@@ -0,0 +1,21 @@
+# stress-ng
+
+> Load and stress a Linux system in various ways (CPU, memory, I/O, etc.).
+> Useful for benchmarking, hardware validation, and stability testing.
+> More information: <https://manned.org/stress-ng>.
+
+- Stress all CPUs with 4 workers for 60 seconds:
+
+`stress-ng {{[-c|--cpu]}} 4 {{[-t|--timeout]}} 60s`
+
+- Stress virtual memory with 2 workers for 30 seconds:
+
+`stress-ng {{[-m|--vm]}} 2 --vm-bytes {{512M}} {{[-t|--timeout]}} 30s`
+
+- Stress the I/O subsystem with 3 workers for 45 seconds:
+
+`stress-ng {{[-i|--io]}} 3 {{[-t|--timeout]}} {{45s}}`
```

Comment:
```suggestion
`stress-ng {{[-i|--io]}} 3 {{[-t|--timeout]}} 45s`
```
Since the time is not explicitly in the description

Reply:
Done.